torchgeo/datasets/xview.py

```diff
@@ -175,8 +175,8 @@ def _load_target(self, path: str) -> Tensor:
         filename = os.path.join(path)
         with Image.open(filename) as img:
             array: "np.typing.NDArray[np.int_]" = np.array(img.convert("L"))
-            tensor: Tensor = torch.from_numpy(array)  # type: ignore[attr-defined]
-            tensor = tensor.to(torch.long)  # type: ignore[attr-defined]
+            tensor = torch.from_numpy(array)
```

Comment:
I usually use `.long()` or (equivalently `.to(torch.int64)`)

Reply:
Let's do that in a separate PR, trying not to change functionality too much in this PR